### ISSUE TITLE
no warnings test may fail

### ISSUE THREAD

**Issue by eserte** [semifor/Twitter-API#21]
On some of my smokers:
```
...
WWW::Form::UrlEncoded::XS 0.25 is require. fallback to PP version at /usr/perl5.18.4p/lib/site_perl/5.18.4/amd64-freebsd/WWW/Form/UrlEncoded.pm line 17.

#   Failed test 'no (unexpected) warnings (via done_testing)'
#   at t/trait/migration.t line 214.
# Looks like you failed 1 test of 28.
t/trait/migration.t .............................. 
Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/28 subtests 
...
```

So possibly this happens if the XS version for WWW::Form::UrlEncoded is not available, or is too old.

**Comment by semifor** [comment]
Looks like we've tried dealing with this before in 33f16aa. Looks to me like I'm setting the `$ENV{WWW_FORM_URLENCODED_PP}` too late.

@eserte can you rerun the test with `WWW_FORM_URLENCODED_PP=1`? If that fixes it, I'll ensure we're setting the environment variable in a BEGIN block before Twitter::API is used.

**Comment by semifor** [comment]
Fixed in 4e787b9, included in release 1.0005.
